ecs-logs-collector.sh

```diff
@@ -549,6 +555,24 @@ enable_docker_debug() {
 
         ok
 
+      fi
+      ;;
+    amazon2)
+
+      if [ -e /etc/sysconfig/docker ] && grep -q "^\s*OPTIONS=\"-D" /etc/sysconfig/docker
+      then
+        info "Debug mode is already enabled."
+      else
+
+        if [ -e /etc/sysconfig/docker ]; then
+          sed -i 's/^OPTIONS="\(.*\)/OPTIONS="-D \1/g' /etc/sysconfig/docker
+
+          try "restart Docker daemon to enable debug mode"
+          /sbin/service docker restart
```

Comment:
Restart on AL2 should use `systemctl` to interact with the `docker.service`, like:

```suggestion
          systemctl restart docker.service
```

Reply:
It also works with service but I agree.